src/main/java/seedu/sudohr/model/SudoHr.java

```diff
@@ -178,6 +194,106 @@ public void removeEmployeeFromDepartment(Person p, Department d) {
 
     //// event-level operations
 
+    /**
+     * Adds a event to the address book.
+     * The event must not already exist in the address book.
+     */
+    public void addEvent(Event event) {
+        events.addEvent(event);
```

Comment:
When adding Event, should check if the user inside the Event exists inside SudoHr.UniquePersonList. I'll need to fix this logic for Department as well. This applies particularly when loading JSON from storage

Reply:
@kennethk-1201  @jerrrren Yeah, i'll have a method for querying if a user exists (by id). Like before, use hashmap?

Reply:
As we will be using internalList, we can stick to brute force approach